site/source/docs/porting/simd.rst

```diff
@@ -1123,47 +1126,47 @@ For detailed information on each intrinsic function, refer to `NEON Intrinsics R
    * - vabdl
      - ⚫ Not implemented, will trigger compiler error
    * - vabs
-     - native
+     - ✅ native
    * - vadd
-     - native
+     - ✅ native
    * - vaddl
      - ⚫ Not implemented, will trigger compiler error
    * - vaddlv
      - ⚫ Not implemented, will trigger compiler error
    * - vaddv
      - ⚫ Not implemented, will trigger compiler error
-   * - vaddw 
+   * - vaddw
      - ❌ Will be emulated with slow instructions, or scalarized
-   * - vand 
-     - native
+   * - vand
+     - ✅ native
    * - vbic
      - ⚫ Not implemented, will trigger compiler error
    * - vbsl
-     - native
+     - ✅ native
    * - vcagt
-     - ⚠ Does not have direct implementation, but is emulated using fast NEON instructions
+     - ⚠️ Does not have direct implementation, but is emulated using fast NEON instructions
```

Comment:
I don't see any visual difference between these lines. Is the emoji different but similar or something?

Reply:
at least on my browser, it was: <img width="251" alt="Screen Shot 2021-09-22 at 8 37 04 AM" src="https://user-images.githubusercontent.com/1749303/134375297-7a165b7a-793e-4a64-a76e-70307c71fac6.png">

now it is: <img width="226" alt="Screen Shot 2021-09-22 at 8 37 39 AM" src="https://user-images.githubusercontent.com/1749303/134375438-05b1899b-482f-45f7-8e92-95716e7e961b.png">

Reply:
The bytes are different:

was: e2  9a a0 0a
now: e2 9a a0 ef b8 8f 

Think it has to do with the yellow fill?